spec/services/acuant/responses/get_results_response_spec.rb

```diff
@@ -0,0 +1,58 @@
+require 'rails_helper'
+
+require 'rails_helper'
+
+describe Acuant::Responses::GetResultsResponse do
+  context 'with a successful result' do
+    subject(:response) do
+      http_response = instance_double(Faraday::Response)
+      allow(http_response).to receive(:body).and_return(AcuantFixtures.get_results_response_success)
+      described_class.new(http_response)
```

Comment:
We can. The reason I didn't was that if we do that we have to use the request classes to init these instead of calling init on them directly. We can get a bit more coverage going through the request objects though so I can do that.

Reply:
I guess what bugged me is the `allow(...).to receive`

I think I'd be just as happy with:

```
http_response = instance_double(Faraday::Response, body: AcuantFixtures.get_results_response_success)`
```
which is basically the same as what's here